docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -119,12 +117,56 @@ During a high-availability backup, the entire YugabyteDB Anywhere state is copie
 
 All instances involved in high availability should be of the same YugabyteDB Anywhere version. If the versions are different, an attempt to promote a standby instance using a YugabyteDB Anywhere backup from an active instance may result in errors.
 
-Even though you can perform an upgrade of all YugabyteDB Anywhere instances simultaneously and there are no explicit ordering requirements regarding upgrades of active and standby instances, it is recommended to follow these guidelines: 
+Even though you can perform an upgrade of all YugabyteDB Anywhere instances simultaneously and there are no explicit ordering requirements regarding upgrades of active and standby instances, it is recommended to follow these general guidelines: 
 
 - Start an upgrade with an active instance.
 - After the active instance has been upgraded, ensure that YugabyteDB Anywhere is reachable by logging in and checking various pages.
 - Proceed with upgrading standby instances. 
 
+The following is the detailed upgrade procedure:
+
+1. Stop the high-availability synchronization. This ensures that only backups of the original YugabyteDB Anywhere version are synchronized to the standby instance.
+
+2. Upgrade the active instance. 
+
+   Expect a momentary lapse in availability for the duration of the upgrade.
+
+3. If upgrade completes, proceed to step 4. Otherwise, proceed to step 9.
+
+4. On the upgraded instance, perform post-upgrade validation tests that may include creating or editing a universe, backups, and so on. If validation is successful, proceed to step 5. Otherwise, proceed to step 9.
+
+5. Perform upgrade of the standby instance.
+
+6. Enable high-availability synchronization.
+
+7. Optionally, perform promotion of the standby instance to the latest backup synchronized from the YugabyteDB Anywhere version to which to upgrade.
+
+8. Expect the process to be completed.
+
+9. Decommission the faulty active instance in the active-standby pair.
```

Comment:
Yes, the whole thing needs work. I'm waiting for dev comments.

Reply:
@eugeneckim Could you, please, review the steps and let me know if they've been captured correctly?

Reply:
The step 9 above as worded is accurate.

Reply:
Steps are accurate, suggesting splitting into 2 sections however. First part ends at 7. Then 'If the upgrade failed, do the following:' and start from 1 again at step 9. Because 9 does not follow from 8 or 7.

Reply:
@ddhodge I agree, the sections based on function/scenario does make things more clear.

Reply:
Changed the list formatting